.github/workflows/deploy.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           node-version: 15
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
+        run: yarn install --frozen-lockfile --ignore-engines
```

Comment:
Can you please explain the use of `--ignore-engines` tag?

Reply:
In the updated dependencies during `yarn install` the packages check for the node server version. If a certain version or later is not found It will throw an error. The following error specifically: `error open@8.4.0: The engine "node" is incompatible with this module. Expected version ">=12". Got "10.24.1"`. `--ignore-engines` tag actually allows that node server version checking to be skipped so that packages are installed without any errors

Reply:
One more thing that can be done is that removing node version 10.x from the tests. As above version 12 there should be no errors

Reply:
@GMishx  Please do have a look! I removed the `--ignore-engines` tag but had to remove node version 10.x from the tests as the new updated dependencies are not compatible with the older version of node.

Reply:
Using `--ignore-engines` is not the correct way to solve the issue. The changes looks better now.